site/cypress/e2e/Cypress-Gradeable/edit_gradeable.spec.js

```diff
@@ -151,7 +151,9 @@ describe('Tests cases revolving around modifying gradeables', () => {
         cy.get('#add-peer-grader').click();
         cy.get('.form-button-container > a').click({ force: true, multiple: true });
 
-        cy.get('#page_5_nav').click();
+        //clicking on page 5 directly, loads the page too fast, which causes flakyness
+        //so we better visit the url
+        cy.visit(['sample', 'gradeable', 'open_peer_homework', 'update?nav_tab=5']);
```

Comment:
I believe you could still test the link by first asserting that the link is visible, and then clicking on it.

Reply:
doesn't cy.visit already do that by default?

Reply:
Cy.visit() doesn't check the links that are on the page, it just tries to visit a URL.

Reply:
I tried to put ```cy.get('#page_5_nav').should('be.visible')```  followed by ```cy.get('#page_5_nav').click()``` or         ```cy.get('#page_5_nav').as('page5');  cy.get('@page5').click();``` but it is still flaky